docs/tutorialkit.dev/astro.config.ts

```diff
@@ -13,6 +13,11 @@ export default defineConfig({
       'Cross-Origin-Opener-Policy': 'same-origin',
     },
   },
+  vite: {
+    ssr: {
+      noExternal: '@tutorialkit/react',
+    },
+  },
```

Comment:
What does this change do?

Reply:
Without this, locally run `pnpm run docs` crashes when react components page is opened in browser.

Reply:
Woow I did noticed that once but thought it might have been my local environment having a bug!! 

Really good find! :star_struck: 

Thanks for the explanation :+1: